src/components/WeatherWidget/WeatherWidget.test.tsx

```diff
@@ -41,6 +51,10 @@ const mockWeatherWidgetWithIncorrectZipcode: WeatherWidgetType = {
 }
 
 describe('WeatherWidget', () => {
+  mockedAxios.get.mockImplementation(() => {
+    return Promise.resolve({ data: mockHourlyForecast })
+  })
+
```

Comment:
I think this might be why that one unit test is failing. The test takes `mockWeatherWidgetWithIncorrectZipcode` which intentionally contains an invalid url, but I think this mocked api call is giving the test data. Maybe you need to just move the test into another `describe`?

Reply:
Thanks for the tip, this made it easier to find!